pkg/db/context.go

```diff
@@ -2,11 +2,41 @@ package db
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/getsentry/sentry-go"
+	"github.com/google/uuid"
 	dbContext "gitlab.cee.redhat.com/service/sdb-ocm-example-service/pkg/db/db_context"
 	"gitlab.cee.redhat.com/service/sdb-ocm-example-service/pkg/logger"
 )
 
+type (
+	contextKey      int
+	advisoryLockMap map[string]*AdvisoryLock
+	LockType        string
+)
+
+const (
+	transactionKey  contextKey = iota
+	advisoryLockKey contextKey = iota + 1
+
+	QuotaLock   LockType = "quota"
+	Migrations  LockType = "migrations"
+	RegcredLock LockType = "regcred"
+	ReplaceLock LockType = "replacement"
```

Comment:
we will use the `migrations` lock for DB migrations.  we don't need these others. they are leftover from AMS.

Reply:
yeah, I left the `migrations` and added a type for `dinosaurs` as an example